lib/Dialect/FIRRTL/Transforms/LowerTypes.cpp

```diff
@@ -207,7 +212,7 @@ static bool isNotSubAccess(Operation *op) {
   if (!sao)
     return true;
   ConstantOp arg = dyn_cast_or_null<ConstantOp>(sao.getIndex().getDefiningOp());
-  return arg && sao.getInput().getType().getNumElements() != 0;
+  return arg && sao.getInput().getType().get().getNumElements() != 0;
 }
 
 /// Look through and collect subfields leading to a subaccess.
```

Comment:
On line 486, need to handle alias type, suggestion,
```
Value TypeLoweringVisitor::getSubWhatever(Value val, size_t index) {
  auto t = val.getType();

  if (firrtl::type_isa<BaseTypeAliasType>(t))
    t = firrtl::type_cast<BaseTypeAliasType>(t)
            .getInnerType()
            .getAnonymousType();

  if (BundleType bundle = t.dyn_cast<BundleType>()) {
    return builder->create<SubfieldOp>(val, index);
  }
  if (FVectorType fvector = t.dyn_cast<FVectorType>()) {
    return builder->create<SubindexOp>(val, index);
  }
  if (t.isa<RefType>()) {
    return builder->create<RefSubOp>(val, index);
  }
  llvm_unreachable("Unknown aggregate type");
  return nullptr;
}
```

Reply:
Yes, thanks